bot.js

```diff
@@ -1,114 +1,59 @@
-      client.me(msg.channelName, `PAJAS 🚨 CUNTS`);
-    };
-
-    // 477589350 = slchbot
-    if (msg.senderUserID === `477589350` && msgText === `pepegasit nevermind`) {
-      client.say(msg.channelName, `alazymDank Slapp slchbot`)
-    };
-
-    if (msg.senderUserID === `477589350` && msgText === `pepea pajbot`) {
-      client.say(msg.channelName, `monkaS it's coming`)
-    };
-
-    if (msgText.includes(`feelsokaybot`)) {
-      client.say(msg.channelName, `pajaBing`);
-    };
-
-    if (msgText.split(" ")[0] === `test` && (msg.channelID === pajladaID || msg.channelID === config.ownerID)) {
-      client.say(msg.channelName, `KKarrot test complete KKarrot`)
-    };
-
-    // Commands with prefixes
-    if (prefixExists) {
-      const noPrefix = stripPrefix.toLowerCase(); // Take the stripped message and convert to lowercase
-      const command = noPrefix.split(' ')[0]; // Take the command name
-      const noCommand = noPrefix.split(' ').slice(1) // Remove the command
-
-      if (command === `ping`) {
-        client.say(msg.channelName, `alazymDank 🏓 ppHop 🏓 MrDestructoid`);
-      };
-
-      if (command === `echo` && msg.senderUserID === config.ownerID) {
-        client.say(msg.channelName, stripPrefix.replace(/^echo/gi,``));
-      };
-
-      if (command === 'pingthemods' && msg.senderUserID === config.ownerID) {
-        let mods = await client.getMods(`#${msg.channelNamel}`);
-        console.log(mods);
-      };
-
-      if (command === 'pingthevips' && msg.senderUserID === config.ownerID) {
-        let vips = await client.getVips(`#${msg.channelNamel}`);
-        console.log(vips);
-      };
-
-      if (command === 'spam' && msg.senderUserID === config.ownerID) {
-        var spamTimes = noCommand.pop()
-        var spamText = noCommand.splice(0).join(" ")
-        while (spamTimes > 0) {
-          client.say(msg.channelName, spamText);
-          spamTimes - 1
-        }
-      };
-    };
-
-    // User is now in 5 second bot timeout
-    talkedRecently.add(msg.senderUserID);
-    setTimeout(() => {
-      // Removes the user from the set after 5 seconds
-      talkedRecently.delete(msg.senderUserID);
-    }, cooldown);
-  };
-});
 
-client.on(`connect`, () => {
-  client.say(config.startupChannel, `Walking!`)
+client.on("ready", () => console.log("Successfully connected to chat"));
+client.on("close", (e) => { if () console.error("[ERR]: Failed to destroy the client.\n",e) });
+client.on("PRIVMSG", (msg) => console.log(`[#${msg.channelName}] ${msg.displayName}: ${msg.messageText}`));
+
+client.on("message", async (msg) => {
+	const prefixExists = String(msg.messageText).starsWith(config.prefix);
+	const stripPrefix = String(msg.messageText).subString(1);
+	const msgText = String(msg.messageText).toLowerCase();
+	const cooldown = config.defaultCooldown;
+	
+	if (msg.senderUserID === config.botID) return;
+	if (talkedRecently.has(msg.sendUserID) && !(msg.sendUserID === config.ownerID)) return; else {
```

Comment:
Your chance of calling this better code died the moment you put an `else {` at the end of a line